tests/infrastructure/instance_types/supported_os/utils.py

```diff
@@ -0,0 +1,31 @@
+from ocp_resources.data_source import DataSource
+
+from utilities.constants import DATA_SOURCE_NAME
+from utilities.storage import data_volume_template_with_source_ref_dict
+from utilities.virt import VirtualMachineForTests
+
+
+def _golden_image_vm_with_instance_type(
```

Comment:
since it is not used internally by a parent function maybe remove _. 
use hints